packages/vue3/tests/unit/lazyload.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { AdvancedImage, lazyload } from "../../src/index";
+import { CloudinaryImage } from "@cloudinary/url-gen/assets/CloudinaryImage";
+import { mount } from "@vue/test-utils";
+import { waitTicks, testWithMockedIntersectionObserver } from "./utils";
+
+const cloudinaryImage = new CloudinaryImage(
```

Comment:
can this be extracted to some common utils?

Reply:
extracted testWithMockedIntersectionObserver